src/test/java/com/github/kovaku/user/GetAllUserTest.java

```diff
@@ -0,0 +1,51 @@
+package com.github.kovaku.user;
+
+import org.json.JSONException;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
```

Comment:
I think this is not used, use Ctrl+Alt+O to optimize imports in Intellij Idea.

Reply:
Your were right, enabled "Optimize imports on the fly" in intellij settings.